packages/react-native/ReactCommon/react/renderer/components/view/ViewShadowNode.cpp

```diff
@@ -62,6 +62,7 @@ void ViewShadowNode::initialize() noexcept {
       viewProps.importantForAccessibility != ImportantForAccessibility::Auto ||
       viewProps.removeClippedSubviews || viewProps.cursor != Cursor::Auto ||
       !viewProps.filter.empty() ||
+      !viewProps.backgroundImage.empty() ||
```

Comment:
This is not the only prop guilty of this, but I think this one should go in `formsView`, instead of `formsStackingContext. I.e. we don't want views with a background image to be flattened, but they also shouldn't form their own stacking context for z-ordering (at least on browsers https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_positioned_layout/Understanding_z-index/Stacking_context)